code/SimpleLight.dm

```diff
@@ -1,8 +1,21 @@
-atom
+/obj/overlay/simple_light
+	event_handler_flags = IMMUNE_SINGULARITY
+	anchored = 2
+	mouse_opacity = 0
+	layer = LIGHTING_LAYER_BASE
+	plane = PLANE_LIGHTING
+	blend_mode = BLEND_ADD
+	icon = 'icons/effects/overlays/simplelight.dmi'
+	icon_state = "3x3"
+	appearance_flags = RESET_COLOR | RESET_TRANSFORM | RESET_ALPHA | NO_CLIENT_COLOR | KEEP_APART
+	pixel_x = -32
+	pixel_y = -32
+
+/atom
 	var
 		list/simple_light_rgbas = null
 
-		mutable_appearance/simple_light = null
+		obj/overlay/simple_light/simple_light = null
```

Comment:
Can you change these to be in the form `/atom/var/foo/bar = baz`, SpacemanDMM is liable to pick this up as a definition of atom, which is annoying for going to the definition.